### ISSUE TITLE
Validation loss is not decreasing on WMT17 EN-ZH 

### ISSUE THREAD

**Issue by suyuzhang** [pytorch/fairseq#3329]
Hi,

I want to train a Transformer-base model for En-Zh translation.
The transformer-base model was trained with the following scripts:

`fairseq-train $DATADIR \
    --source-lang en --target-lang zh --weight-decay 0.0001 \
    --clip-norm 0.0 --max-tokens 4096 --ddp-backend=no_c10d \
    --optimizer adam --adam-betas '(0.9, 0.98)' \
    --dropout 0.1 --attention-dropout 0.1 --label-smoothing 0.1 \
    --warmup-updates 4000 --warmup-init-lr 1e-07 \
    --lr 0.0005 --lr-scheduler inverse_sqrt --max-update 300000 \
    --criterion label_smoothed_cross_entropy \
    --keep-interval-updates 10 --save-interval-updates 10000 \
    --keep-last-epochs 10 --log-interval 100 \
    --arch transformer_wmt_en_de --save-dir $TRAIN \
    --tensorboard-logdir $TRAIN/logdir`

WMT17 EN-ZH preprocessing:
[https://github.com/twairball/fairseq-zh-en/blob/master/wmt17_prepare.sh](https://github.com/twairball/fairseq-zh-en/blob/master/wmt17_prepare.sh)

I find the validation loss is not decreasing.
Here is train and validation loss graph:
![loss_log](https://user-images.githubusercontent.com/28804865/110424315-8f958280-80dd-11eb-8736-ba6e1ec0775e.PNG)

What is wrong?
Thanks!

**Comment by tensorfoo** [comment]
hi @suyuzhang how did you produce those graphs from the hydra log?

**Comment by haorannlp** [comment]
Maybe nothing is wrong here, just overfitting

**Comment by SJTUer-yuheng** [comment]
You need a large batch size for En-Zh. @suyuzhang 
Maybe max-tokens 4096 update-freq 16
